docs/guides/upgrading.rst

```diff
@@ -93,6 +93,7 @@ All the functions in ``engine/lib/deprecated-1.9.php`` were removed. See https:/
 All the functions in ``engine/lib/deprecated-1.10.php`` were removed. See https://github.com/Elgg/Elgg/blob/2.0/engine/lib/deprecated-1.10.php for these functions. Each ``@deprecated`` declaration includes instructions on what to use instead.
 
  * ``_elgg_manage_pagesetup``
+ * ``_elgg_clear_entity_files``: Use ``ElggEntity->clearAllFiles()``
```

Comment:
It was private, do we need it in the docs?

Reply:
i suggest we make the entity function `protected` than maybe only mention it (no alternative)